packages/vsce/src/trees/ResourceInspectorViewProvider.ts

```diff
@@ -212,4 +218,41 @@ export class ResourceInspectorViewProvider implements WebviewViewProvider {
       nonce,
     });
   }
+
+  /**
+   * Handles the showLogsForHyperlink request from the webview
+   * Fetches region data directly using runGetResource and calls showRegionLogs command
+   */
+  private async handleShowLogsForHyperlink() {
+    const { regionName, cicsplexName, profileName } = this.resourceContext;
+    try {
+      const { response } = await runGetResource({
+        profileName,
+        resourceName: CicsCmciConstants.CICS_CMCI_REGION,
+        regionName,
+        cicsPlex: cicsplexName,
+      });
+
+      if (response.records?.cicsregion) {
+        const regionRecords = toArray(response.records.cicsregion);
+        if (regionRecords.length > 0) {
+          const regionData = regionRecords[0];
+          if(regionData && regionData.jobid){
+              const jobid = regionData.jobid;  
+              const cicsProfile = SessionHandler.getInstance().getProfile(profileName);
+              await findProfileAndShowJobSpool(cicsProfile, jobid, regionName);
+            }
+          } else {
+              CICSLogger.debug(`Empty region records array for ${regionName}`);
+              window.showErrorMessage(l10n.t("Could not find region data and job id for region {0} to show logs.", regionName));
+          }
+      } else {
+          CICSLogger.debug(`No region records found for ${regionName}`);
+          window.showErrorMessage(l10n.t("Could not find any record for region {0} to show logs.", regionName));
+      }
+    } catch (error) {
+        CICSLogger.error(`Error showing logs for hyperlink: ${error.message}`);
```

Comment:
i think we should put the region name in
```suggestion
        CICSLogger.error(`Error showing logs for hyperlink. Region: ${regionName}, Error: ${error.message}`);
```

will error.message lose the stack trace? in which case is there a better way to log the whole error?